Core/Security/X509Certificate.py

```diff
@@ -252,3 +276,13 @@ def getExtensions( self ):
         value = "Cannot decode value"
       extList.append( ( sn, value ) )
     return S_OK( sorted( extList ) )
+
+  def verify( self, pkey ):
+    ret = self.__certObj.verify( pkey )
+    return S_OK( ret )
+
+  def get_subject( self ):
+    return self.getSubjectDN()['Value']
```

Comment:
OK, will fix that. I guess it's one of the hacks to allow other parts (SSLSockets, etc.) work in kind of legacy mode. I tried not to use SSL-related object calls like get_subject outside of X509Certificate class (use getSubject -> S_OK instead), but some parts of code expected pyGSI X509 objects, not Core.Security.X509Certificate.